pages/index.js

```diff
@@ -1,15 +1,13 @@
 import Head from "next/head";
 import { useRouter } from "next/router";
-import { useState } from 'react';
 import { siteTitle } from "../components/Layout";
 import {
   getAuth,
-  signInWithEmailAndPassword,
-  createUserWithEmailAndPassword,
-  onAuthStateChanged,
-  signOut
+  onAuthStateChanged
```

Comment:
What's this import of the auth state monitor doing here when you have commented out the invocation of it in the code inside the Home component? Would you explain to me how does it work and what it does and doesn't do by calling it here in the Home component versus not calling it here?

Reply:
Agreed. I think it is related to the component where the user lists are being fetched (Sidebar/TodoListsNav?)